README.md

```diff
@@ -203,18 +203,14 @@ Did everything work as expected? Great - now onto the production site.
 Make sure your changes are deployed to the production server (i.e. you've cut a release with your changes).
 
 1. Look at the times cron tasks are run (specified in [`scripts/lametro-crontasks`](https://github.com/datamade/la-metro-councilmatic/blob/master/scripts/lametro-crontasks)), and plan to rebuild the index inbetween those times. Rebuilding the index will take a few minutes, so plan accordingly.
-2. Open a new pull request and comment out the crons under [`scripts/lametro-crontasks`](https://github.com/datamade/la-metro-councilmatic/blob/master/scripts/lametro-crontasks).
-3. Merge in the pull request and deploy.
-4. As above: shell into the server as the ubuntu user, go to the `lametro` repo, and remove and rebuild the Solr container.
-5. Switch to the datamade user and rebuild the index.
+2. Merge in the pull request and deploy.
+3. As above: shell into the server, go to the `lametro` repo, then remove and rebuild the Solr container.
```

Comment:
I think it would be useful to provide the commands, since you have to account for the production-version of Solr, i.e., the container is `lametro-staging-solr`, and the command is `sudo docker-compose up -d solr-staging`. Being explicit seems best, to me.